tauri-examples/hello_world/src-tauri/setup-ios.sh

```
@@ -0,0 +1,62 @@
+
+if [ -d "$XCFRAMEWORK" ]; then
+  echo "sherpa-onnx.xcframework already exists at $XCFRAMEWORK"
+  exit 0
+fi
+
+URL="https://github.com/k2-fsa/sherpa-onnx/releases/download/xcframework/sherpa-onnx-v${VERSION}-ios-shared-onnxruntime-static.xcframework.zip"
+echo "Downloading sherpa-onnx xcframework v${VERSION}..."
+echo "  $URL"
+
+TMPFILE=$(mktemp /tmp/sherpa-onnx-XXXXXX.zip)
+trap "rm -f $TMPFILE" EXIT
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)